### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by khajabakshi** [datapane/datapane#36] [Ideas]
Hello Team!

Hope you are doing well!!

I'm using Datapane library from last 2 weeks, it is fantastic. I'm impressed with the formats and methods included for plots. But deep inside I'm not satisfied with the Tables. 

I feel like Tables should have more customization options like "**trend icon indicator**". 

Adding this type of features to tables, this library will become one stop destination for all sort of amazing visual and statistical approaches.

Hope you consider this feature request as valid and push an update very soon. Thank you!

Good luck!!!

**Comment by lanthias** [comment]
Thanks for the feature request! 

We will definitely be improving our table. One question I have: for this sort of custom table, we are going to be adding an HTML block. This would allow you to style a DataFrame using `pandas` and include it in your report using the `df.to_html` method. 

This is extremely flexible, in that you could generate the trend icon using Python, as is demonstrated in [this StackOverflow question](https://stackoverflow.com/questions/52247052/python-show-html-arrows-to-dataframe). Would that be a suitable solution to your use-case? 

Thanks a lot!
Leo

**Comment by khajabakshi** [comment]
Thanks for looking into my feature request!

I've checked the thread from the link provided by you, I'd say YES!! It would address my use case. But I'd like to mention a small improvement to that. Instead of creating another column for indicating the trend icons try to include in the same column beside the data.

Really appreciate your efforts in addressing and developing the new feature. Hope to see the new feature very soon.

Thanks,
